From c972841191070e1cabb646e30f06a174ffecf5ec Mon Sep 17 00:00:00 2001
From: Pavel Grunt <pgrunt@redhat.com>
Date: Mon, 27 Feb 2017 13:47:39 +0100
Subject: [PATCH virt-viewer] Comment out folder sharing menus
To: virt-tools-list@redhat.com

Folder sharing requires phodav which is not available in el7
Do not display menu items related to it.

Related: rhbz#1413982
Signed-off-by: Eduardo Lima (Etrunko) <etrunko@redhat.com>
---
 src/resources/ui/virt-viewer.ui |  9 ---------
 src/virt-viewer-window.c        | 13 -------------
 2 files changed, 22 deletions(-)

diff --git a/src/resources/ui/virt-viewer.ui b/src/resources/ui/virt-viewer.ui
index e9609ec..a1070c8 100644
--- a/src/resources/ui/virt-viewer.ui
+++ b/src/resources/ui/virt-viewer.ui
@@ -82,15 +82,6 @@
                           </object>
                         </child>
                         <child>
-                          <object class="GtkMenuItem" id="menu-preferences">
-                            <property name="visible">True</property>
-                            <property name="can_focus">False</property>
-                            <property name="label" translatable="yes">_Preferences</property>
-                            <property name="use_underline">True</property>
-                            <signal name="activate" handler="virt_viewer_window_menu_preferences_cb" swapped="no"/>
-                          </object>
-                        </child>
-                        <child>
                           <object class="GtkSeparatorMenuItem" id="separatormenuitem1">
                             <property name="visible">True</property>
                             <property name="can_focus">False</property>
diff --git a/src/virt-viewer-window.c b/src/virt-viewer-window.c
index 867a7b0..5997a37 100644
--- a/src/virt-viewer-window.c
+++ b/src/virt-viewer-window.c
@@ -63,7 +63,6 @@ void virt_viewer_window_menu_file_usb_device_selection(GtkWidget *menu, VirtView
 void virt_viewer_window_menu_file_smartcard_insert(GtkWidget *menu, VirtViewerWindow *self);
 void virt_viewer_window_menu_file_smartcard_remove(GtkWidget *menu, VirtViewerWindow *self);
 void virt_viewer_window_menu_view_release_cursor(GtkWidget *menu, VirtViewerWindow *self);
-void virt_viewer_window_menu_preferences_cb(GtkWidget *menu, VirtViewerWindow *self);
 void virt_viewer_window_menu_change_cd_activate(GtkWidget *menu, VirtViewerWindow *self);
 
 
@@ -316,7 +315,6 @@ virt_viewer_window_init (VirtViewerWindow *self)
     gtk_widget_set_sensitive(GTK_WIDGET(gtk_builder_get_object(self->priv->builder, "menu-send")), FALSE);
     gtk_widget_set_sensitive(GTK_WIDGET(gtk_builder_get_object(self->priv->builder, "menu-view-zoom")), FALSE);
     gtk_widget_set_sensitive(GTK_WIDGET(gtk_builder_get_object(self->priv->builder, "menu-file-screenshot")), FALSE);
-    gtk_widget_set_sensitive(GTK_WIDGET(gtk_builder_get_object(self->priv->builder, "menu-preferences")), FALSE);
 
     gtk_builder_connect_signals(priv->builder, self);
 
@@ -967,13 +965,6 @@ virt_viewer_window_menu_file_smartcard_remove(GtkWidget *menu G_GNUC_UNUSED,
 }
 
 G_MODULE_EXPORT void
-virt_viewer_window_menu_preferences_cb(GtkWidget *menu G_GNUC_UNUSED,
-                                       VirtViewerWindow *self)
-{
-    virt_viewer_app_show_preferences(self->priv->app, self->priv->window);
-}
-
-G_MODULE_EXPORT void
 virt_viewer_window_menu_view_release_cursor(GtkWidget *menu G_GNUC_UNUSED,
                                             VirtViewerWindow *self)
 {
@@ -1282,9 +1273,6 @@ virt_viewer_window_set_menus_sensitive(VirtViewerWindow *self, gboolean sensitiv
 
     priv = self->priv;
 
-    menu = GTK_WIDGET(gtk_builder_get_object(priv->builder, "menu-preferences"));
-    gtk_widget_set_sensitive(menu, sensitive);
-
     menu = GTK_WIDGET(gtk_builder_get_object(priv->builder, "menu-file-screenshot"));
     gtk_widget_set_sensitive(menu, sensitive);
 
@@ -1373,7 +1361,6 @@ virt_viewer_window_set_display(VirtViewerWindow *self, VirtViewerDisplay *displa
         if (virt_viewer_display_get_enabled(display))
             virt_viewer_window_desktop_resize(display, self);
 
-        gtk_widget_set_sensitive(GTK_WIDGET(gtk_builder_get_object(self->priv->builder, "menu-preferences")), TRUE);
         gtk_widget_set_sensitive(GTK_WIDGET(gtk_builder_get_object(self->priv->builder, "menu-view-zoom")), TRUE);
         gtk_widget_set_sensitive(GTK_WIDGET(gtk_builder_get_object(self->priv->builder, "menu-send")), TRUE);
         gtk_widget_set_sensitive(self->priv->toolbar_send_key, TRUE);
-- 
2.12.0

